tests/lib/cli.js

```diff
@@ -887,6 +928,35 @@ describe("cli", () => {
 
         });
 
+        it("should pass fixTypes to CLIEngine when --fix-type is passed", () => {
```

Comment:
Is this a duplicate of the test starting on line 778?

Reply:
It certainly appears that way. Nice catch! I'll remove it.